althea_kernel_interface/src/setup_wg_if.rs

```diff
@@ -124,6 +124,42 @@ impl dyn KernelInterface {
         }
         Ok(timestamps)
     }
+
+    /// Returns the last handshake time of every ACTIVE client on this tunnel.
```

Comment:
you should describe what active means here, essentially any device that has handshaked since the tunnel was created